crates/libcontainer/src/rootfs/utils.rs

```diff
@@ -243,7 +245,7 @@ mod tests {
         )?;
         assert_eq!(
             MountOptionConfig {
-                flags: MsFlags::MS_NOSUID,
+                flags: MsFlags::MS_NOSUID | MsFlags::MS_STRICTATIME,
```

Comment:
May I ask why you changed this unit test? I just want your intention.

Reply:
Did you fix it?
https://github.com/youki-dev/youki/pull/2937#issuecomment-2542820780

Reply:
Yes. IIUC, the original test code seemed wrong. It tried to parse the following mount options.

```
// ...
            &MountBuilder::default()
                .destination(PathBuf::from("/dev"))
                .typ("tmpfs")
                .source(PathBuf::from("tmpfs"))
                .options(vec![
                    "nosuid".to_string(),
                    "strictatime".to_string(),
                    "mode=755".to_string(),
                    "size=65536k".to_string(),
                ])
                .build()?,
// ...
```

It specifies both `strictatime` and `nosuid`, but the original test code's expected result was only `MsFlags::MS_NOSUID`.

If I misunderstood the spec, please let me know and I'll check my code again to fix the CI.